src/components/Spine.mdx

```diff
@@ -0,0 +1,243 @@
+---
+name: 'Spine'
+route: '/Spine'
+menu: Components
+---
+
+# Spine Component
+
+The component implements the [pixi-spine plugin](https://github.com/pixijs/pixi-spine) which enables the Spine support, including some nice extra features to make easier to work with dynamic content with Spine. 
+
+```js
+ class Pixie extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      animation: 'running'
+    }
+  }
+
+  componentDidMount() {
+    const loader = PixiLoader.shared;
+    loader
+        .add('pixie', 'public/spine/pixie.json')
+        .load((ld, res) => {
+          this.setState({
+            resource: res.pixie.spineData
+          })
+        });
+  }
+  
+  animationEnd = (event, track) => {
+    console.log(event)
+    console.log(track)
+  }
+
+  animationStart = (event, track) => {
+    if(event.animation.name === 'jump') {
+      this.setState(prevState => {
+        return {
+          animation: 'running'
+        }
+      })
+    }
+  }
+
+
+  onclick = (el) => {
+    this.setState({
+      animation: 'jump'
+    })
+  }
+
+  render() {
+    const { resource, animation } = this.state
+    return (
+      <Stage width={1024} height={1024} resolution = {window.devicePixelRatio || 1} options={{ backgroundColor: 0x012b30}}>
+         <Container x={520} y={500}>
+           {resource && (
+              <SpineContainer 
+              	mixes={[{from: 'running', to: 'jump', time: 0.2}, {from: 'jump', to: 'running', time: 0.4}]} 
+              	animations={[animation]}  
+              	x={0} 
+              	y={400}
+              	scale={0.5} 
+              	pointerup={this.onclick} 
+              	interactive 
+              	buttonMode 
+              	resource={spineData} 
+              	events={
+                  {
+                    complete: this.animationEnd, // internal event
+                    start: this.animationStart // internal event
+                    halfwayJump: this.halfwayJump // custom timeline event
+                  }
+                 }>
+                   <SpineContainer.Animation name={'jump'} play={false} track={0} />
```

Comment:
Personally I prevent `<X.Y>` syntax, can we rename the component. 
For instance:

```jsx
<SpineContainer>
<SpineAnimation>
<SpineSlot>
<SpineBone>
```

etc